### PR TITLE
Corrected Typos and Grammar

### PR DESCRIPTION
**Pull request by cs-rohit-khune**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ Navigate to demo alert and note the following:
     - Next steps
 
 
-- The playbook `Fortinet FortiNDR Cloud > Fetch Detection Events` runs manually and performs following actions.
+- The playbook `Fortinet FortiNDR Cloud > Fetch Detection Events` runs manually and performs the following actions.
 
     - Fetch events for specified detection and append the same to the source data in the alert.
 
-    - Creates a record for the asset on which data exfiltration issue detected and links the same to the alert.
+    - Creates a record for the asset on which the data exfiltration issue was detected and links the same to the alert.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Suggestion: Creates a record for an asset on which data exfiltration issue is detected and link it to an alert.

File: docs/usage.md [hunk 2]
```diff
@@ -27,14 +27,14 @@ Navigate to demo alert and note the following:
     - Next steps
 
 
-- The playbook `Fortinet FortiNDR Cloud > Fetch Detection Events` runs manually and performs following actions.
+- The playbook `Fortinet FortiNDR Cloud > Fetch Detection Events` runs manually and performs the following actions.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md:30]: the following actions or the following action?

File: docs/usage.md (hunk 1 above)
Comment (cs-rohit-khune) [reply docs/usage.md]: Update this as below:
Creates a record for an asset on which the data exfiltration issue is detected and links it to the alert.

File: docs/usage.md (hunk 2 above)
Comment (cs-rohit-khune) [reply docs/usage.md:30]: `actions` would be better as playbook performing multiple things

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**